src/main/java/com/minecolonies/core/entity/citizen/EntityCitizen.java

```diff
@@ -48,14 +48,16 @@
 import com.minecolonies.core.colony.jobs.JobKnight;
 import com.minecolonies.core.colony.jobs.JobNetherWorker;
 import com.minecolonies.core.colony.jobs.JobRanger;
-import com.minecolonies.core.entity.SittingEntity;
-import com.minecolonies.core.entity.ai.basic.AbstractEntityAIBasic;
-import com.minecolonies.core.entity.ai.citizen.CitizenAI;
-import com.minecolonies.core.entity.ai.citizen.guard.AbstractEntityAIGuard;
+import com.minecolonies.core.entity.citizen.citizenhandlers.*;
+import com.minecolonies.core.entity.other.SittingEntity;
+import com.minecolonies.core.entity.ai.workers.AbstractEntityAIBasic;
+import com.minecolonies.core.entity.ai.workers.CitizenAI;
+import com.minecolonies.core.entity.ai.workers.guard.AbstractEntityAIGuard;
 import com.minecolonies.core.entity.ai.minimal.EntityAICitizenChild;
 import com.minecolonies.core.entity.ai.minimal.EntityAIInteractToggleAble;
 import com.minecolonies.core.entity.ai.minimal.LookAtEntityGoal;
-import com.minecolonies.core.entity.citizen.citizenhandlers.*;
+import com.minecolonies.core.entity.entity.citizen.citizenhandlers.*;
```

Comment:
These imports look strange to me. Was there already an `entity` package within `entity`?